shenyu-admin/src/main/java/org/apache/shenyu/admin/utils/CommonUpstreamUtils.java

```diff
@@ -87,18 +88,22 @@ public static GrpcUpstream buildDefaultGrpcUpstream(final String host, final Int
     public static TarsUpstream buildDefaultTarsUpstream(final String host, final Integer port) {
         return TarsUpstream.builder().upstreamUrl(buildUrl(host, port)).weight(50).warmup(10).timestamp(System.currentTimeMillis()).build();
     }
-    
+
     /**
      * Convert common upstream list list.
      *
      * @param upstreamList the upstream list
      * @return the list
      */
     public static List<CommonUpstream> convertCommonUpstreamList(final List<? extends CommonUpstream> upstreamList) {
-        return upstreamList.stream().map(upstream -> new CommonUpstream(upstream.getProtocol(), upstream.getUpstreamHost(), upstream.getUpstreamUrl())).collect(Collectors.toList());
+        return Optional.ofNullable(upstreamList)
```

Comment:
when upstreamList can be null？

Reply:
* When shenyu-admin first starting ,
* In UpstreamCheckService.java line 316, converUpstream has error.![2](https://user-images.githubusercontent.com/19170961/150147863-b38158f5-7697-4191-a4cc-c9b78afc85a0.jpeg)
![1](https://user-images.githubusercontent.com/19170961/150147973-b40f0531-333f-4518-8d84-aa39d16e94f5.jpeg)